Makefile

```diff
@@ -192,6 +192,16 @@ codeserver-ubi9-python-3.9: base-ubi9-python-3.9
 intel-base-gpu-ubi9-python-3.9: base-ubi9-python-3.9
 	$(call image,$@,intel/base/gpu/ubi9-python-3.9,$<)
 
+# Build and push intel-runtime-tensorflow-ubi9-python-3.9 image to the registry
+.PHONY: intel-runtime-tensorflow-ubi9-python-3.9
+intel-runtime-tensorflow-ubi9-python-3.9: intel-base-gpu-ubi9-python-3.9
+	$(call image,$@,intel/runtimes/tensorflow/ubi9-python-3.9,$<)
+
+# Build and push jupyter-intel-tensorflow-ubi9-python-3.9 image to the registry
+.PHONY: jupyter-intel-tensorflow-ubi9-python-3.9
+jupyter-intel-tensorflow-ubi9-python-3.9: intel-runtime-tensorflow-ubi9-python-3.9
```

Comment:
Noticed you are building the jupyter one with runtime as base?
any specific reason for this ?

Reply:
Our initial design was to re-use the runtime and only install jupyter dependencies on top. We will update it to separate the runtime and jupyter in the next release as that will be optimal.

Reply:
ack, perhaps
in release, we should make the jupyter one and runtime timee both build from base.
i noticed, the runtime has, openai inclusion,
if that is also required in jupyter one, lets get this in 
and work on the rest in next one

Reply:
Agreed, we can work on building on top of the base image in the next release. The oneAPI basekit is needed for both containers.